inferno/io/transform/volume.py

```diff
@@ -120,7 +126,7 @@ def volume_function(self, volume):
 class VolumeCenterCrop(Transform):
     """ Crop patch of size `size` from the center of the volume """
     def __init__(self, size, **super_kwargs):
-        super(VolumeCrop, self).__init__(**super_kwargs)
+        super(VolumeCenterCrop, self).__init__(**super_kwargs)
```

Comment:
Good catch.
Let's switch to new super syntax though:
`super().__init__(**super_kwargs)`

Reply:
Just saying, literally all the other classes call base class Python2-style :)

Reply:
Yes and it's an eye-sore ;)